microsoft/testsuites/ltp/ltp.py

```diff
@@ -53,29 +55,73 @@ class Ltp(Tool):
     DEFAULT_LTP_TESTS_GIT_TAG = "20230929"
     LTP_GIT_URL = "https://github.com/linux-test-project/ltp.git"
     BUILD_REQUIRED_DISK_SIZE_IN_GB = 2
-    LTP_RESULT_PATH = "/opt/ltp/ltp-results.log"
-    LTP_OUTPUT_PATH = "/opt/ltp/ltp-output.log"
-    LTP_SKIP_FILE = "/opt/ltp/skipfile"
     COMPILE_TIMEOUT = 1800
     RUN_TIMEOUT = 12000
 
     @property
     def command(self) -> str:
-        return "/opt/ltp/runltp"
+        if self._ltp_tar_file:
+            return "/tmp/ltp/runltp"
+        else:
+            return "/opt/ltp/runltp"
+
+    @property
+    def command_dir(self) -> str:
+        if self._ltp_tar_file:
+            return "/tmp/ltp/"
+        else:
+            return "/opt/ltp/"
 
     @property
     def dependencies(self) -> List[Type[Tool]]:
-        return [Make, Gcc, Git]
+        if self._ltp_tar_file:
+            return []
+        else:
+            return [Make, Gcc, Git]
 
     @property
     def can_install(self) -> bool:
         return True
 
-    def __init__(self, node: Node, *args: Any, **kwargs: Any) -> None:
+    @property
+    def result_path(self) -> str:
+        if self._ltp_tar_file:
+            return "/tmp/ltp/ltp-results.log"
+        else:
+            return "/opt/ltp/ltp-results.log"
+
+    @property
+    def output_path(self) -> str:
+        if self._ltp_tar_file:
+            return "/tmp/ltp/ltp-output.log"
+        else:
+            return "/opt/ltp/ltp-output.log"
+
+    @property
+    def skip_path(self) -> str:
+        if self._ltp_tar_file:
+            return "/tmp/ltp/skipfile"
+        else:
+            return "/opt/ltp/skipfile"
+
+    def _check_exists(self) -> bool:
+        return (
+            len(self.node.tools[Ls].list(str(self.command_dir), sudo=True)) > 0
+        )
+
+    def __init__(self, node: Node, ltp_tar_file: str, *args: Any, **kwargs: Any) -> None:
```

Comment:
`ltp_tar_file` doesn't describe the use of the variable. It should be called like `prebuilt_package`.

Reply:
The change is done, thanks